pkg/cmd/roachtest/run.go

```diff
@@ -517,12 +517,11 @@ func runOperation(register func(registry.Registry), filter string, clusterName s
 	op.Status(fmt.Sprintf("operation ran successfully; waiting %s before cleanup", roachtestflags.WaitBeforeCleanup))
 	select {
 	case <-ctx.Done():
-		return ctx.Err()
```

Comment:
thanks for doing this! might be worth adding a comment here that this is to get Ctrl+C to run cleanup.